perception_dataset/deepen/segmentation/painting3d.py

```diff
@@ -0,0 +1,167 @@
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+import json
+import re
+from typing import Any, Dict, List
+
+from perception_dataset.deepen.deepen_annotation import DeepenAnnotation, LabelType
+
+__all__ = [
+    "DeepenSegmentationPainting3DAnnotations",
+    "DeepenSegmentationPainting3DScene",
+    "DeepenSegmentationPainting3DDataset",
+]
+
+
+@dataclass
+class DeepenSegmentationPainting3DScene(DeepenAnnotation):
+    """
+    A class to save a 3D segmentation annotation for a scene annotated by painting.
+    :param paint_categories: List of categories in lidarseg.
+    :param lidarseg_anno_file: Binary annotation file for the lidarseg.
+    :param total_lidar_points: Total number of lidar pointclouds.
+    """
+
+    paint_categories: List[str] = field(default_factory=list)
+    lidarseg_anno_file: str = ""
```

Comment:
If we doesn't allow empty list and string, we shouldn't set their default values.
```suggestion
    paint_categories: List[str]
    lidarseg_anno_file: str
```

Reply:
Make sense, updated